packages/contracts/utils/buildArtifacts.ts

```diff
@@ -45,6 +45,15 @@ export const MinimumViableMultisig = AbstractContract.fromArtifactName(
   }
 );
 
+// tslint:disable-next-line
```

Comment:
y?

Reply:
all of the exports have this - it's because of the non camelCased variable name